seatunnel-core/seatunnel-core-sql/src/main/java/io/github/interestinglab/waterdrop/core/sql/job/Executor.java

```diff
@@ -46,17 +48,30 @@ private static StatementSet handleStatements(String workFlowContent, StreamTable
 
         StatementSet statementSet = tEnv.createStatementSet();
         TableEnvironmentImpl stEnv = (TableEnvironmentImpl) tEnv;
+        Configuration configuration = tEnv.getConfig().getConfiguration();
 
         List<String> stmts = SqlStatementSplitter.normalizeStatements(workFlowContent);
         for (String stmt : stmts) {
             Operation op = stEnv.getParser().parse(stmt).get(0);
             if (op instanceof CatalogSinkModifyOperation) {
                 statementSet.addInsertSql(stmt);
+            } else if (op instanceof SetOperation) {
+                callSetOperation(configuration, (SetOperation) op);
             } else {
                 tEnv.executeSql(stmt);
             }
         }
         return statementSet;
     }
 
+    private static void callSetOperation(Configuration configuration, SetOperation setOperation) {
+
+        // set property
+        if (setOperation.getKey().isPresent() && setOperation.getValue().isPresent()) {
```

Comment:
we can improve here by using `orElseThrow`.

Reply:
Make sense, thanks.